<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EnterInputs</class>
 <widget class="QMainWindow" name="EnterInputs">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>629</width>
    <height>473</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>UML Editor</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/icons/images/new/network-config.png</normaloff>:/icons/images/new/network-config.png</iconset>
  </property>
  <property name="dockOptions">
   <set>QMainWindow::AllowNestedDocks|QMainWindow::AllowTabbedDocks|QMainWindow::AnimatedDocks</set>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="margin">
     <number>0</number>
    </property>
    <item>
     <layout class="QVBoxLayout" name="mainLayout"/>
    </item>
    <item>
     <widget class="QTabWidget" name="datagrams">
      <property name="tabPosition">
       <enum>QTabWidget::North</enum>
      </property>
      <property name="currentIndex">
       <number>-1</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>629</width>
     <height>18</height>
    </rect>
   </property>
   <widget class="QMenu" name="maFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionCreate_project"/>
    <addaction name="actionOpen_project"/>
    <addaction name="actionSave_project"/>
    <addaction name="separator"/>
    <addaction name="actionGeneration"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="maActions">
    <property name="title">
     <string>Actions</string>
    </property>
    <addaction name="actionAdd_activity"/>
    <addaction name="actionAddState"/>
    <addaction name="actionAdd_condition"/>
    <addaction name="actionAdd_sync"/>
    <addaction name="actionAdd_author"/>
    <addaction name="actionAdd_use_case"/>
    <addaction name="actionAdd_node"/>
    <addaction name="actionAdd_module"/>
    <addaction name="actionAdd_coloboration"/>
    <addaction name="actionAdd_sequence"/>
    <addaction name="actionAdd_class"/>
    <addaction name="separator"/>
    <addaction name="actionEdit_name_of_diagram"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionShow_toolbar"/>
   </widget>
   <widget class="QMenu" name="menuDiagrams">
    <property name="title">
     <string>Diagrams</string>
    </property>
    <addaction name="actionCreate_activity_diagram"/>
    <addaction name="actionCreate_fsm_diagram"/>
    <addaction name="actionCreate_use_case_diagram"/>
    <addaction name="actionCreate_class_diagram"/>
    <addaction name="actionCreate_topology_diagram"/>
    <addaction name="actionCreate_coloboration_diagram"/>
    <addaction name="actionCreate_sequence_diagram"/>
   </widget>
   <addaction name="maFile"/>
   <addaction name="maActions"/>
   <addaction name="menuView"/>
   <addaction name="menuDiagrams"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="windowTitle">
    <string>Панель инструментов</string>
   </property>
   <property name="toolTip">
    <string>Панель инструментов</string>
   </property>
   <attribute name="toolBarArea">
    <enum>LeftToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionOpen_project"/>
   <addaction name="actionSave_project"/>
   <addaction name="separator"/>
   <addaction name="actionGeneration"/>
   <addaction name="separator"/>
   <addaction name="actionAdd_activity"/>
   <addaction name="actionAdd_condition"/>
   <addaction name="actionAdd_sync"/>
   <addaction name="actionAddState"/>
   <addaction name="actionAdd_use_case"/>
   <addaction name="actionAdd_author"/>
   <addaction name="actionAdd_node"/>
   <addaction name="actionAdd_module"/>
   <addaction name="actionAdd_sequence"/>
   <addaction name="actionAdd_coloboration"/>
   <addaction name="actionAdd_class"/>
   <addaction name="separator"/>
   <addaction name="actionLevel_up"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionAddState">
   <property name="icon">
    <iconset resource="../../share/mainResources.qrc">
     <normaloff>:/icon/StateBody</normaloff>:/icon/StateBody</iconset>
   </property>
   <property name="text">
    <string>Add state</string>
   </property>
   <property name="shortcut">
    <string>+</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset resource="../../share/mainResources.qrc">
     <normaloff>:/icons/images/exit.png</normaloff>:/icons/images/exit.png</iconset>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionGeneration">
   <property name="icon">
    <iconset resource="../../share/mainResources.qrc">
     <normaloff>:/icons/images/finish.png</normaloff>:/icons/images/finish.png</iconset>
   </property>
   <property name="text">
    <string>Generation</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="actionLevel_up">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="../../share/mainResources.qrc">
     <normaloff>:/icons/images/go-up.png</normaloff>:/icons/images/go-up.png</iconset>
   </property>
   <property name="text">
    <string>Level up</string>
   </property>
   <property name="toolTip">
    <string>Level up</string>
   </property>
   <property name="shortcut">
    <string>PgUp, Backspace</string>
   </property>
  </action>
  <action name="actionSave_project">
   <property name="icon">
    <iconset resource="../../share/mainResources.qrc">
     <normaloff>:/textedit/images/filesave.png</normaloff>:/textedit/images/filesave.png</iconset>
   </property>
   <property name="text">
    <string>Save project</string>
   </property>
   <property name="toolTip">
    <string>Save project</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionOpen_project">
   <property name="icon">
    <iconset resource="../../share/mainResources.qrc">
     <normaloff>:/icons/images/document-open.png</normaloff>:/icons/images/document-open.png</iconset>
   </property>
   <property name="text">
    <string>Open project</string>
   </property>
   <property name="toolTip">
    <string>Open project</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionCreate_project">
   <property name="icon">
    <iconset resource="../../share/mainResources.qrc">
     <normaloff>:/icons/images/document-new.png</normaloff>:/icons/images/document-new.png</iconset>
   </property>
   <property name="text">
    <string>Create project</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionAdd_activity">
   <property name="icon">
    <iconset resource="../../share/mainResources.qrc">
     <normaloff>:/icon/icons/idea.png</normaloff>:/icon/icons/idea.png</iconset>
   </property>
   <property name="text">
    <string>Add activity</string>
   </property>
   <property name="toolTip">
    <string>Add activity</string>
   </property>
   <property name="shortcut">
    <string>+</string>
   </property>
  </action>
  <action name="actionAdd_condition">
   <property name="icon">
    <iconset resource="../../share/mainResources.qrc">
     <normaloff>:/icon/icons/messagebox_warning.png</normaloff>:/icon/icons/messagebox_warning.png</iconset>
   </property>
   <property name="text">
    <string>Add condition</string>
   </property>
   <property name="shortcut">
    <string>+</string>
   </property>
  </action>
  <action name="actionAdd_sync">
   <property name="icon">
    <iconset resource="../../share/mainResources.qrc">
     <normaloff>:/icon/icons/run.png</normaloff>:/icon/icons/run.png</iconset>
   </property>
   <property name="text">
    <string>Add sync</string>
   </property>
   <property name="toolTip">
    <string>Add sync</string>
   </property>
  </action>
  <action name="actionAdd_author">
   <property name="icon">
    <iconset resource="../../share/mainResources.qrc">
     <normaloff>:/icon/UseCaseBody</normaloff>:/icon/UseCaseBody</iconset>
   </property>
   <property name="text">
    <string>Add author</string>
   </property>
   <property name="toolTip">
    <string>Add author</string>
   </property>
  </action>
  <action name="actionAdd_use_case">
   <property name="icon">
    <iconset resource="../../share/mainResources.qrc">
     <normaloff>:/icon/icons/games.png</normaloff>:/icon/icons/games.png</iconset>
   </property>
   <property name="text">
    <string>Add use case</string>
   </property>
  </action>
  <action name="actionAdd_node">
   <property name="icon">
    <iconset resource="../../share/mainResources.qrc">
     <normaloff>:/icon/TopologyBody</normaloff>:/icon/TopologyBody</iconset>
   </property>
   <property name="text">
    <string>Add node</string>
   </property>
   <property name="toolTip">
    <string>Add node</string>
   </property>
  </action>
  <action name="actionAdd_module">
   <property name="icon">
    <iconset resource="../../share/mainResources.qrc">
     <normaloff>:/icon/ClassBody</normaloff>:/icon/ClassBody</iconset>
   </property>
   <property name="text">
    <string>Add module</string>
   </property>
   <property name="toolTip">
    <string>Add module</string>
   </property>
  </action>
  <action name="actionAdd_coloboration">
   <property name="icon">
    <iconset resource="../../share/mainResources.qrc">
     <normaloff>:/icon/CollaborationBody</normaloff>:/icon/CollaborationBody</iconset>
   </property>
   <property name="text">
    <string>Add coloboration</string>
   </property>
   <property name="toolTip">
    <string>Add coloboration</string>
   </property>
  </action>
  <action name="actionShow_toolbar">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../../share/mainResources.qrc">
     <normaloff>:/icon/icons/run.png</normaloff>:/icon/icons/run.png</iconset>
   </property>
   <property name="text">
    <string>Show toolbar</string>
   </property>
   <property name="toolTip">
    <string>Show toolbar</string>
   </property>
  </action>
  <action name="actionAdd_sequence">
   <property name="icon">
    <iconset resource="../../share/mainResources.qrc">
     <normaloff>:/icon/SequenceBody</normaloff>:/icon/SequenceBody</iconset>
   </property>
   <property name="text">
    <string>Add sequence</string>
   </property>
  </action>
  <action name="actionAdd_class">
   <property name="icon">
    <iconset resource="../../share/mainResources.qrc">
     <normaloff>:/icon/icons/file-manager.png</normaloff>:/icon/icons/file-manager.png</iconset>
   </property>
   <property name="text">
    <string>Add class</string>
   </property>
  </action>
  <action name="actionCreate_activity_diagram">
   <property name="icon">
    <iconset resource="../../share/mainResources.qrc">
     <normaloff>:/icon/ActivityBody</normaloff>:/icon/ActivityBody</iconset>
   </property>
   <property name="text">
    <string>Create activity diagram</string>
   </property>
  </action>
  <action name="actionCreate_fsm_diagram">
   <property name="icon">
    <iconset resource="../../share/mainResources.qrc">
     <normaloff>:/icon/StateBody</normaloff>:/icon/StateBody</iconset>
   </property>
   <property name="text">
    <string>Create fsm diagram</string>
   </property>
  </action>
  <action name="actionCreate_coloboration_diagram">
   <property name="icon">
    <iconset resource="../../share/mainResources.qrc">
     <normaloff>:/icon/CollaborationBody</normaloff>:/icon/CollaborationBody</iconset>
   </property>
   <property name="text">
    <string>Create coloboration diagram</string>
   </property>
  </action>
  <action name="actionCreate_sequence_diagram">
   <property name="icon">
    <iconset resource="../../share/mainResources.qrc">
     <normaloff>:/icon/SequenceBody</normaloff>:/icon/SequenceBody</iconset>
   </property>
   <property name="text">
    <string>Create sequence diagram</string>
   </property>
  </action>
  <action name="actionCreate_class_diagram">
   <property name="icon">
    <iconset resource="../../share/mainResources.qrc">
     <normaloff>:/icon/ClassBody</normaloff>:/icon/ClassBody</iconset>
   </property>
   <property name="text">
    <string>Create class diagram</string>
   </property>
  </action>
  <action name="actionCreate_topology_diagram">
   <property name="icon">
    <iconset resource="../../share/mainResources.qrc">
     <normaloff>:/icon/TopologyBody</normaloff>:/icon/TopologyBody</iconset>
   </property>
   <property name="text">
    <string>Create topology diagram</string>
   </property>
  </action>
  <action name="actionCreate_use_case_diagram">
   <property name="icon">
    <iconset resource="../../share/mainResources.qrc">
     <normaloff>:/icon/UseCaseBody</normaloff>:/icon/UseCaseBody</iconset>
   </property>
   <property name="text">
    <string>Create use case diagram</string>
   </property>
  </action>
  <action name="actionEdit_name_of_diagram">
   <property name="icon">
    <iconset resource="../../share/mainResources.qrc">
     <normaloff>:/icon/icons/pencil.png</normaloff>:/icon/icons/pencil.png</iconset>
   </property>
   <property name="text">
    <string>Edit name of diagram</string>
   </property>
   <property name="toolTip">
    <string>Edit name of active diagram</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../../share/mainResources.qrc"/>
 </resources>
 <connections/>
</ui>
